### PR TITLE
Fix max db conns in iris, atreugo, go-std; add atreugo-gojay, atreugo-easyjson, atreugo-sjson

### PR DESCRIPTION
**Pull request by chunariov**

### CODE REVIEW

File: frameworks/Go/atreugo/src/handlers/handlers_gojay.go [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+	}
+}
+
+// QueriesHandlerGoJay . Test 3: Multiple database queries
+func QueriesHandlerGoJay(db storage.DB) func(ctx *atreugo.RequestCtx) error {
+	return func(ctx *atreugo.RequestCtx) error {
+		queries := queriesParam(ctx)
+
+		// worlds := make([]storage.World, queries)
+		worlds := storage.WorldsPool.Get().(storage.Worlds)[:queries]
```
Comment (nbrady-techempower) [inline frameworks/Go/atreugo/src/handlers/handlers_gojay.go]: Think you meant `(storage.World)` here

File: frameworks/Go/atreugo/src/handlers/handlers_gojay.go [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+	}
+}
+
+// UpdateHandlerGoJay . Test 5: Database updates
+func UpdateHandlerGoJay(db storage.DB) func(ctx *atreugo.RequestCtx) error {
+	return func(ctx *atreugo.RequestCtx) error {
+		queries := queriesParam(ctx)
+		var err error
+
+		worlds := storage.WorldsPool.Get().(storage.Worlds)[:queries]
```
Comment (nbrady-techempower) [inline frameworks/Go/atreugo/src/handlers/handlers_gojay.go]: and here

File: frameworks/Go/atreugo/src/handlers/handlers_gojay.go (hunk 1 above)
Comment (chunariov) [reply frameworks/Go/atreugo/src/handlers/handlers_gojay.go]: Thanks

File: frameworks/Go/atreugo/src/handlers/handlers_gojay.go (hunk 2 above)
Comment (chunariov) [reply frameworks/Go/atreugo/src/handlers/handlers_gojay.go]: Thanks

**Comment by chunariov** [comment]
There are errors not in my commits. How I can avoid them?

**Comment by nbrady-techempower** [comment]
@chunariov All the Go/revel tests are failing and you touched them here

**Comment by nbrady-techempower** [comment]
@chunariov I cannot merge this until the revel tests pass. Can you remove the revel changes from this PR if you don't intend to test/fix them?

**Comment by nbrady-techempower** [comment]
@chunariov please keep revel in Travis. We don’t want to remove tests from Travis unless we remove them from the repo and I don’t want to remove revel from the repo right now. I’ll ping the maintainer of revel to see if they want to fix it if it’s broken in its current state.

**Comment by chunariov** [comment]
OK. I will create new pull request from clean fork and ONLY with my changes and Travis tests.

**Pull request closed without merging**